conda/plan.py

```diff
@@ -290,6 +290,9 @@ def ensure_linked_actions(dists, prefix, index=None, force=False, always_copy=Fa
         if not extracted_in:
             actions[inst.EXTRACT].append(dist)
 
+        # find_new_location returns either a `pkgs` directory or the filename.
+        if not fetched_in.endswith('.tar.bz2'):
+            fetched_in = join(fetched_in, install._dist2filename(dist))
```

Comment:
Yeah, I thought that may be the case, but I didn't want to change something called from so many sites.